package.json

```diff
@@ -50,13 +50,17 @@
     "src"
   ],
   "dependencies": {
-    "@loopback/context": "^0.12.2",
     "@loopback/boot": "^0.12.2",
+    "@loopback/context": "^0.12.2",
     "@loopback/core": "^0.11.2",
     "@loopback/dist-util": "^0.3.5",
+    "@loopback/openapi-v3": "^0.12.2",
     "@loopback/repository": "^0.14.2",
     "@loopback/rest": "^0.19.2",
-    "@loopback/openapi-v3": "^0.12.2"
+    "@types/bcrypt": "^2.0.0",
+    "bcrypt": "^3.0.0",
```

Comment:
Please use https://github.com/dcodeIO/bcrypt.js instead.

Reply:
`bcrypt` is a binary add-on and it requires compilation.

Reply:
And we don't want that? CI was failing before because I was running install with `--ignore-scripts`. The JS version is about 30% slower according to their own docs.

Reply:
Plain JS module is much more desirable for an example app. It will remove a lot of support issues in the future.